docs/contributing-more.md

```diff
@@ -0,0 +1,21 @@
+# Contributing, Continued
```

Comment:
Couldn't this just go into `CONTRIBUTING.md` since that file was only 20 lines, or maybe the file name should be about development environment, however some isn't.

Not relevant for your change but that CONTRIBUTING document is quite stale :(, I guess the reason I kept the Castle Core one nice and simple: https://github.com/castleproject/Core/blob/master/CONTRIBUTING.md